### PR TITLE
New js-packages, metrics-engine http endpoint

### PR DESCRIPTION
**Pull request by pastorsx**
generic (js-package directory)
- added metrics-engine and gis-engine in js-packages

metrics engine
- addition of a http endpoint to query metrics-engine db to pull the logs that were saved for http  requests and notifications

platform ctrl
- addition of the name of the sandbox that got created by the platform rest api when creating a sandbox with autogenerated name (name also returned if the name was provided in endpoint)

metrics store
- improvement in the getInflux functionality to take a list of param as tag to combine with an OR in the query to prevent multiple requests

### CODE REVIEW

File: js-apps/meep-frontend/package-lock.json [hunk 1]
```diff
@@ -9953,7 +9958,7 @@
       "requires": {
         "is-in-browser": "^1.1.3",
         "symbol-observable": "^1.1.0",
-        "warning": "^4.0.1"
+        "warning": "^3.0.0"
```
Comment (dilallkx) [inline js-apps/meep-frontend/package-lock.json:9961]: The "warning" package must be version ^4.0.1. Update all references from 3.0.0 to 4.0.1

File: .meepctl-repocfg.yaml [hunk 2]
```diff
@@ -108,6 +108,8 @@ repo:
           meep-platform-ctrl-api: js-packages/meep-platform-ctrl-client
           meep-sandbox-ctrl-api: js-packages/meep-sandbox-ctrl-client
           meep-mon-engine-api: js-packages/meep-mon-engine-client
+          meep-metrics-engine-api: js-packages/meep-metrics-engine-client
```
Comment (dilallkx) [inline .meepctl-repocfg.yaml:111]: meep-frontend does not use this package. Please remove this reference.

File: .meepctl-repocfg.yaml [hunk 3]
```diff
@@ -316,3 +318,7 @@ repo:
       meep-sandbox-ctrl-client:
         src: js-packages/meep-sandbox-ctrl-client
         lint: false
+      meep-metrics-engine-client:
```
Comment (dilallkx) [inline .meepctl-repocfg.yaml:321]: Minor comment: packages are listed in alphabetical order.

File: go-packages/meep-metric-store/metric-store.go [hunk 4]
```diff
@@ -266,11 +266,20 @@ func (ms *MetricStore) GetInfluxMetric(metric string, tags map[string]string, fi
...
+			tagStr = " WHERE (" // + k + "='" + v + "'"
 		} else {
-			tagStr += " AND " + k + "='" + v + "'"
+			tagStr += " AND (" //+ k + "='" + v + "'"
+		}
+                for i, v := range mv {
+			if i != 0 {
+				tagStr += " OR "
+			}
+              		tagStr += k + "='" + v + "'"
```
Comment (dilallkx) [inline go-packages/meep-metric-store/metric-store.go:280]: Fix indentation.

File: go-packages/meep-metric-store/metric-store.go [hunk 5]
```diff
@@ -266,11 +266,20 @@ func (ms *MetricStore) GetInfluxMetric(metric string, tags map[string]string, fi
 	// Tags
 	tagStr := ""
 	for k, v := range tags {
+		mv := strings.Split(v, ",")
```
Comment (dilallkx) [inline go-packages/meep-metric-store/metric-store.go:269]: I recommend adding a UT for this new functionality.

File: js-apps/meep-frontend/package-lock.json (hunk 1 above)
Comment (pastorsx) [reply js-apps/meep-frontend/package-lock.json:9961]: done

File: .meepctl-repocfg.yaml (hunk 2 above)
Comment (pastorsx) [reply .meepctl-repocfg.yaml:111]: done

File: .meepctl-repocfg.yaml (hunk 3 above)
Comment (pastorsx) [reply .meepctl-repocfg.yaml:321]: reordered

File: go-packages/meep-metric-store/metric-store.go (hunk 4 above)
Comment (pastorsx) [reply go-packages/meep-metric-store/metric-store.go:280]: fixed

File: go-packages/meep-metric-store/metric-store.go (hunk 5 above)
Comment (pastorsx) [reply go-packages/meep-metric-store/metric-store.go:269]: added ut test for 2 loggers in httplog-test

**Pull request merged**